webapp/components/create_post.jsx

```diff
@@ -322,10 +322,8 @@ export default class CreatePost extends React.Component {
         document.addEventListener('keydown', this.showShortcuts);
     }
 
-    componentDidUpdate(prevProps, prevState) {
-        if (prevState.channelId !== this.state.channelId) {
-            this.focusTextbox();
-        }
+    componentDidUpdate() {
+        this.focusTextbox();
```

Comment:
I can't get onto the test server, but I'm almost certain this will keep grabbing keyboard focus as files finish uploading and possibly as modals open and close. Can you test these cases? I'm really leery about doing stuff on every update since you never know what will trigger them